stdlib/concurrent/futures/process.pyi

```diff
@@ -1,26 +1,22 @@
 import multiprocessing as mp
-import multiprocessing.connection
+import multiprocessing.connection as mpconn
+import multiprocessing.context as mpcont
```

Comment:
Just use `from abc import Whatever`, unless an import conflicts with a name in this module.